src/test-container.ts

```diff
@@ -29,6 +29,8 @@ export interface TestContainer {
   withDefaultLogDriver(): this;
   withPrivilegedMode(): this;
   withPullPolicy(pullPolicy: PullPolicy): this;
+  withCopyFileToContainer(sourcePath: string, containerPath: string): this;
+  withCopyContentToContainer(content: string | Buffer | Readable, containerPath: string): this;
```

Comment:
What do you think about creating a `Content` type? `type Content = string | Buffer | Readable`

Reply:
Hmm, I'm concerned that this will make the API less readable. For instance here's how the IDE behaves currently:

![image](https://user-images.githubusercontent.com/6221386/105832118-3a3f5f00-5fd0-11eb-87b0-cf871705ddd0.png)

Where as with the type alias it will look like this:

![image](https://user-images.githubusercontent.com/6221386/105833472-e2a1f300-5fd1-11eb-80b7-3f3886a5de39.png)

The second screenshot doesn't really tell what can I actually pass as the argument, and I have to go and look up the definition of `Content`. In the first screenshot it is immediately clear that I am free to pass strings, buffers and readable streams. Does that make sense?

Reply:
You're right, definitely better without the type, I hadn't considered the lack of IntelliSense, thanks!